tests/spec/test_dagspec.py

```diff
@@ -1884,6 +1885,41 @@ def test_lazy_load_product_level_client(tmp_directory, tmp_imports,
     assert 'my_testing_module' in sys.modules
 
 
+@pytest.mark.parametrize('client_spec', [
+    'my_testing_module.get_db_client',
+])
+@pytest.mark.parametrize(
+    'tasks, expected',
+    [([{
+        'source': 'script.sql',
+        'product': 'another.csb',
+        'client': 'my_testing_module.get_db_client',
+        'product_client': 'my_testing_module.get_db_client'
+    }], "'.csb' is not a valid extension. Did you mean: .csv"),
+     ([{
+         'source': 'script.sql',
+         'product': 'another.parquets',
+         'client': 'my_testing_module.get_db_client',
+         'product_client': 'my_testing_module.get_db_client'
+     }], "'.parquets' is not a valid extension. Did you mean: .parquet")])
+def test_product_extension_typo(tasks, expected, tmp_directory, client_spec):
+    Path('script.sql').write_text("""
+    SELECT * FROM my_table
+    """)
+
+    with sqlite3.connect('my.db') as conn:
```

Comment:
i think you can simplify this test. since the `DAGSpec` call will fail, ploomber will never actually check if the data exists or not. I think if you remove this, the test will still be ok

Reply:
Removed this